modules/annotations/src/main/java/org/mule/config/expression/ExpressionFilterParser.java

```diff
@@ -26,6 +27,14 @@
  */
 public class ExpressionFilterParser
 {
+
+    private MuleContext muleContext;
+
+    public ExpressionFilterParser(MuleContext muleContext)
```

Comment:
Is obvious that it creates an ExpressionFilterParser instance, but what about the context? is required or can be null?